README.md

```diff
@@ -1,24 +1,24 @@
-# Opam Repository for Citrix Hypervisor
+# Opam Repository for the XenServer Hypervisor
```

Comment:
It's not exactly xenserver, since it's also used for xcp-ng, mentioning xapi as well is more specific:
```suggestion
# Opam Repository for the xapi components of xenserver
```

Reply:
Is it `xenserver` or `XenServer`?